### DISCUSSION TITLE
Fibargroup FGS213 and FGS223 - "Turn On" behaviour when mode is "AUTO OFF"

### DISCUSSION THREAD

**Discussion by candrea77** [zwave-js/zwave-js#6652] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
When a relay is cofigured as "Auto OFF" (which means that it automatically turn off after xx seconds) the behaviour is :
1) - Initial state : relay turned off
2) - Click Turn On --> relay turn on
3) - Click again Turn On --> relay turn off

What did you expect to happen instead?
1) - Initial state : relay turned off
2) - Click Turn On --> relay turn on
3) - Click again Turn On --> relay remain on
For example , fgs-221 with "Auto On" have the correct behaviour, or also the same fgs-223 without "Auto On" is working ok

1) - Configure a relay (number 1 or number 2) with "AUTO OFF" feature
2) - Click Turn On
3) - Click Turn On

Anything else we should know?
If FGS223 is not setup with "AUTO OFF" , the issue is not present

### Software versions

zwave-js-ui: 9.8.1.04f1c76
zwave-js: 12.4.3

This behaviour is present in the zwave-js ui and also in home assistant

### Device information

Manufacturer: Fibaro
Model name: FGS-223
Node ID: 126
0x010f 0x0203-0x1000

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

Starting from 2024-02-01T23:22:32.298Z DRIVER » [Node 126] [REQ] [SendDataBridge]

Try a couple of "Turn On" on ID 126 and then on ID 128
- 126 is supposed to have the issue (FGS-223 with AUTO ON on relay 1)
- 128 is supposed to haven't the issue (FGS-223 without AUTO ON on relay 1)
[zwavejs_2024-02-02.log](https://github.com/zwave-js/node-zwave-js/files/14132151/zwavejs_2024-02-02.log)

**Comment by AlCalzone** [comment]
Z-Wave JS tells the device to turn on. If the device turns off in response to that, it's a device bug.

Configuration parameters are meaningless to Z-Wave JS, they are just numbers. The implementation of the auto-on/off feature is entirely on the device.

**Reply by candrea77** [reply]
Hi,
clear to me (I was thinking the same thing, just needed a confirmation)

**Comment by candrea77** [comment, marked as answer]
And obviously this was all my faults 
The behaviour is adjustable with the parameter 11 ( for relay 1 ) and 16 (for relay 2) "First Channel - Reaction to Key S1 for Delay/Auto ON/OFF Modes"

We can leave this discussion in case someone else run into same behaviour.
